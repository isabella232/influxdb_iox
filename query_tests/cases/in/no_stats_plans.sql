-- Demonstrate plans that are not optimized using statistics
-- IOX_SETUP: OldTwoMeasurementsManyFieldsOneRubChunk

-- This plan should scan data as it reads from chunk with delete predicates
EXPLAIN SELECT count(*) from h2o;

-- NOTE: This test should have "IOX_SETUP: ThreeDeleteThreeChunks" but becasue of  Bug: https://github.com/influxdata/influxdb_iox/issues/2745
-- make it OldTwoMeasurementsManyFieldsOneRubChunk
-- Also, the query should be "EXPLAIN SELECT count(*) from cpu;"

