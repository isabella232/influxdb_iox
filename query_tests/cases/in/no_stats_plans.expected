-- Test Setup: OldTwoMeasurementsManyFieldsOneRubChunk
-- SQL: EXPLAIN SELECT count(*) from h2o;
+---------------+-------------------------------------------------------------+
| plan_type     | plan                                                        |
+---------------+-------------------------------------------------------------+
| logical_plan  | Projection: #COUNT(UInt8(1))                                |
|               |   Aggregate: groupBy=[[]], aggr=[[COUNT(UInt8(1))]]         |
|               |     TableScan: h2o projection=Some([0])                     |
| physical_plan | ProjectionExec: expr=[COUNT(UInt8(1))@0 as COUNT(UInt8(1))] |
|               |   ProjectionExec: expr=[3 as COUNT(UInt8(1))]               |
|               |     EmptyExec: produce_one_row=true                         |
|               |                                                             |
+---------------+-------------------------------------------------------------+
